### DISCUSSION TITLE
how to keep the bookmark list open

### DISCUSSION THREAD

**Discussion by uvwildos** [mbnuqw/sidebery#918] [Q&A] [answered]
its a very nice tool.
it would be even better if I could keep the bookmark list open, instead of having it switch to the tab list all the time.
is there a setting i could not find?

if not this is a feature request ;)

**Comment by mbnuqw** [comment, marked as answer]
You can open config for bookmarks panel (right-click on the bookmark panel icon in Sidebery sidebar > Configure panel or open a settings page and click on that bookmark panel in the list of panels). There, you can toggle the "Prevent auto-switching from this panel" option.

**Reply by uvwildos** [reply]
indeed that I didnt find yet. 
thanks a lot
